src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -31,9 +31,8 @@
                         <td colspan="2" class="title">
                             <img src="https://demo.sylius.com/assets/shop/img/logo.png" style="width:300px;">
                         </td>
-
                         <td>
-                            {{ 'sylius_refund.ui.issued_at'|trans }}: {{ creditMemo.issuedAt|date('Y-m-d H:i:s') }}<br>
+                            {{ 'sylius_refund.ui.issued_at'|trans }}: {% if creditMemo.issuedAt is not null %}{{ creditMemo.issuedAt|date('Y-m-d H:i:s') }}{% endif %}<br>
```

Comment:
I've noticed, that `CreditMemo->getIssuedAt()` has not nullable return typehint, therefore this check is unnecessary here and in other places.

So doctrine mapping for CreditMemo issuedAt needs to updated to non-nullable and getIssuedAt method should clone the value or work with immutable datatime.